sat-extension/src/main/java/org/openhab/tools/analysis/report/SummaryReportHtmlGenerator.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.tools.analysis.report;
+
+import static org.openhab.tools.analysis.report.ReportUtil.*;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.channels.FileChannel;
+import java.nio.channels.FileLockInterruptionException;
+import java.text.MessageFormat;
+import java.time.Duration;
+import java.time.Instant;
+
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+
+import org.codehaus.plexus.component.annotations.Component;
+import org.codehaus.plexus.component.annotations.Requirement;
+import org.codehaus.plexus.logging.Logger;
+
+import com.google.common.io.Files;
```

Comment:
Can't this be `java.nio.file.Files`?

Reply:
I also wasn't happy with this and tried changing them (it's code from the original Mojo) but for some reason it didn't work well so I left it as is. I'll see if I can conquer this in a second attempt.

Reply:
For unknown reasons it seems to work fine now (Java 11).